apollo/src/main/graphql/com/hedvig/android/owldroid/graphql/home.graphql

```diff
@@ -73,4 +73,23 @@ query HomeQuery($locale: Locale!, $languageCode: String!) {
     }
     body
   }
+
+  quoteBundle(input: $quoteBundleInput) {
```

Comment:
This whole query now fails if we dont supply a list of ids here, should we ask back-end to handle that error better? Perhaps return some sort of `QuoteBundleResult` instead of `QuoteBundleInput`?